manifests/params.pp

```diff
@@ -52,14 +54,121 @@
   # no client certs should be trusted for auth by default.
   $ssl_certs_dir          = undef
 
-  if ($::operatingsystem == 'Ubuntu' and $::lsbdistrelease == '10.04') or ($::operatingsystem == 'SLES') {
+  if ($::apache::globals::scl) {
+    $_scl_httpd = $::apache::globals::scl_httpd
+    $_scl_php_version = $::apache::globals::scl_php_version
+    $verify_command = "/opt/rh/${_scl_httpd}/root/usr/sbin/apachectl -t"
+  } elsif ($::operatingsystem == 'Ubuntu' and $::lsbdistrelease == '10.04') or ($::operatingsystem == 'SLES') {
     $verify_command = '/usr/sbin/apache2ctl -t'
   } elsif $::operatingsystem == 'FreeBSD' {
     $verify_command = '/usr/local/sbin/apachectl -t'
   } else {
     $verify_command = '/usr/sbin/apachectl -t'
   }
-  if $::osfamily == 'RedHat' or $::operatingsystem =~ /^[Aa]mazon$/ {
+  if ($::apache::globals::scl) {
```

Comment:
This is a RH specific technology so I'd think it should be an if clause within the RH osfamily. Also allows you to reduce a lot of variable duplication.

Reply:
It is RedHat that provides it, but I think that it is very different from normal RH software.

Therefore I would like to hold scl separate from normal RedHat-config as it will be fewer lines, but each line will be twice as complex to understand. And changes to SCL will then have the possibility to affect RedHat configuration.